src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -84,6 +85,27 @@
         return emails;
     }
     
+    /**
+     * Generates the feedback submission confirmation email for the given {@code session} for {@code student}
+     * and {@code instructor}.
+     */
+    public EmailWrapper generateFeedbackSubmissionConfirmationEmails(
+            FeedbackSessionAttributes session, StudentAttributes student,
+            InstructorAttributes instructor) {
+        CourseAttributes course = coursesLogic.getCourse(session.getCourseId());
+        EmailWrapper email = new EmailWrapper();
+        
+        if (instructor != null) {
+            email = generateFeedbackSessionEmailBaseForInstructorSubmissionConfirmation(course, session,
+                    instructor);
+        }
+        if (student != null) {
+            email = generateFeedbackSessionEmailBaseForStudentSubmissionConfirmation(course, session,
+                    student);
+        }
+        return email;
```

Comment:
The first one is a wrong assumption as a person can be both an instructor and student in the same course. Of course, if that is the case, then either one of the emails is valid.
The second one is exactly my concern. At the call site of this method, is it possible that both `instructor` and `student` are null?